packages/ag-charts-community/src/module/optionsGraph.ts

```diff
@@ -1105,4 +1115,59 @@ export class OptionsGraph extends AdjacencyListGraph<unknown, string> implements
         this.rollbackEdgesValue = [];
         this.isRollingBack = false;
     }
+
+    private predictAxes(seriesType: SeriesType, userSeriesOptions?: any, data?: DatumDefault[]) {
```

Comment:
Could we easily extract this to a utility function, this doesn't feel like the best place for this logic to live?